libs/ui/search/src/lib/results-table/results-table.component.html

```diff
@@ -120,4 +121,27 @@
       {{ dateToString(item.recordUpdated) }}
     </ng-template>
   </gn-ui-interactive-table-column>
+
+  <!-- ACTION MENU COLUMN -->
+  <gn-ui-interactive-table-column>
+    <ng-template #header> </ng-template>
+    <ng-template #cell let-item>
+      <button
```

Comment:
do you think you could use a gn-ui-button there? to have some hover/focus effects (the stop propagation part will also be handled that way)

Reply:
I had tried, but I'll try again.
Opening the menu didn't seem to work with a gn-ui-button, maybe because of the stop propagation indeed.